app/views/users/logix/index.html.erb

```diff
@@ -43,7 +43,11 @@
 
   <% if @user.projects.blank? %>
     <div class ="no-projects px-4">
-      <h5>You don't have any projects. Create one <%= link_to 'here', simulator_new_url, class: 'no-projects-here' %></h5>
+      <% if @current_user.id.eql? @user.id %>
+        <h5>You don't have any projects. Create one <%= link_to 'here', simulator_new_url, class: 'no-projects-here' %></h5>
+      <% else %>
+        <h5><%= @user.name %> don't have any projects</h5>
```

Comment:
doesn't

Reply:
didn't get your point @nitin10s

Reply:
@Nitish145, @nitin10s means to say that Please change don't to doesn't in the else block

Reply:
Damn, my bad :(